Backpack/Rating/Classes/BPKRating.m

```diff
@@ -32,6 +32,9 @@ @interface BPKRating ()
 @property(nonatomic) BPKRatingTextWrapper *textWrapper;
 @property(nonatomic) BPKRatingBubble *ratingBubble;
 @property(nonatomic) NSLayoutConstraint *textSpacingConstraintHorizontal;
+@property(nonatomic) NSLayoutConstraint *textSpacingConstraintVertical;
```

Comment:
do we have these `textSpacingConstraintVertical` and `textSpacingConstraintHorizontal` separated in properties rather than contained inside the `horizontalLayoutConstraints` or `verticalLayoutConstraints` by some reason?

Reply:
We should be - fixed now 👍